### PR TITLE
FUCK POCKETS

### PR DESCRIPTION
**Pull request by Cheridan**
Mostly a revert of #19587 Removes suit pockets.
oh my god i hate this feature

Want to take off your suit? Oops, instead it opened up its inventory, closing the bag you had open.
There's no indication of what suits have pockets or don't, it's just more bullshit that you have to memorize. 
Completely inconsistent with the rest of clothing, which is all removed when you click on it.

An incredibly aggravating interface issue for the sake of storage creep.

### CODE REVIEW

**Comment by YakumoChen** [comment]
i ded

**Comment by YakumoChen** [comment]
just change it so click drag or shift click opens pockets or something

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-//Bartender //it's chef what the fuck is your problem?
```
Comment (Iamgoofball) [inline code/modules/clothing/head/jobs.dm]: No

**Comment by TechnoAlchemisto** [comment]
Honestly? I never use the suit pockets because the one thing I would use them for would be for internals and they don't work for that. But yeah the stripping shit is dumb, and they really don't have a use, because pockets are for easy access , and these are anything but.

It's cool to have the framework though. It should just be used very, very sparingly.

**Comment by Cheridan** [comment]
Yeah if @Core0verload wants to make a pr that adds:
A. Some form of indication that something has pockets
B. Better interface controls
then it'd be fine. But it's his feature and also he makes buttbaby reverts all the time so I'm doing it to him :hocho:

**Comment by xxalpha** [comment]
:+1: but what about making it an alt click function to open the pockets instead?

**Comment by Core0verload** [comment]
I make reverts of shit changes no one but you wants. Looks like someone's butt blasted about the recent one.

I'll do something with pockets if there would be something *concrete* for me to do. Not "some form" or "better".

**Comment by Cheridan** [comment]
I literally told you what to fix. It needs an indication of what items have pockets, and it needs a better interface. I don't care how you go about it, it's your feature to fix.

**Comment by XDTM** [comment]
Alt-click to open would also fix being unable to use slime potions on them, maybe

**Comment by WJohn** [comment]
Alt clicking to open anything is friggin' unacceptable. Pretty much nothing in the game outside of checking what's on a tile or vent crawling even uses alt clicking.

**Comment by CosmicScientist** [comment]
>just change it so click drag or shift click opens pockets or something

Click drag would be consistent with every other container you can carry or wear. This would also fix how you have to wear a bloody suit to get at the contents when you don't for anything else. And whilst I like having more space to store the millions of items I have as a detective, it's no fun to forget any of it exists.
>Alt clicking to open anything is friggin' unacceptable. Pretty much nothing in the game outside of checking what's on a tile or vent crawling even uses alt clicking.

Locking lockers. Whilst it's become second nature to me now, it still looks like an awkward hurdle for new players to get over and it feels wildly out of place... and using an item in hand on a closed locker locks it which is frustrating but then I'm complaining about something else. I don't want the inconsistent meme of Alt+click to be used in _very_ different ways when it's not an _alternative_ use for something that doesn't already have an existing method.

**Comment by Jalleo** [comment]
We really need to put in the examine text of all items of what alt click does on it.

**Comment by WJohn** [comment]
See I didn't even know alt clicking lockers to lock them was even a thing until you mentioned it. It's too rarely used to justify it, and should either have its uses completely removed or vastly expanded so it's everywhere.

**Comment by kevinz000** [comment]
OOC: Desucake: The only fuck pocket is your groin tbh

**Comment by PKPenguin321** [comment]
I hide emags and beacons in my hardsuit all the time

**Comment by lzimann** [comment]
pockets are a pain, please merge this

**Comment by Aloraydrel** [comment]
Bye bye

**Comment by matillo** [comment]
Just change to `priority = FALSE` on selected items. Even seems to be DESIGNED for noob discovery rather than an actual permanent setting.

> priority = TRUE // so the detectives would discover pockets in their hats

File: code/modules/clothing/head/jobs.dm (hunk 1 above)
Comment (ike709) [inline code/modules/clothing/head/jobs.dm]: undocumented change

Comment (lzimann) [inline code/modules/clothing/head/jobs.dm]: rip legendary comment

**Comment by Cheridan** [comment]
@matillo The game is hard enough for new players to get into without intentionally cultivating basic systems that are obtuse and hidden, while experienced players are running around with two free slots to put brute patches or whatever in.

**Comment by nullbear** [comment]
suit pockets arent even typically useful as storage. Best use i've seen is as a hidden storage, because nobody thinks to check them.

**Comment by Core0verload** [comment]
Would it be better if I would make pockets open only on click-drag and show "It has pockets" on examine?

**Comment by Cheridan** [comment]
That would do

**Comment by optimumtact** [comment]
cheri does a thing

**Comment by optimumtact** [comment]
While the refactoring is underway this can safely be merged as it's fairly easy to undo and is clearly a pain point for the players.

**Pull request merged**